### PR TITLE
Address Feature/enhancement78 and bug in creation of pipelines

### PR DESCRIPTION
**Pull request by Baronhez**
This pull request fixes two problems. 

- Now it is possible to execute pipeline_generator.sh from any directory. Closes https://github.com/devonfw/hangar/issues/78

- A bug related to the creation of pipelines is solved by using two flags.

### CODE REVIEW

**Comment by Baronhez** [comment]
e

**Comment by Baronhez** [comment]
Now, changes were made to make the code maintenance easier.

File: scripts/pipelines/azure-devops/templates/package/quarkus-package.sh [hunk 1]
```diff
@@ -39,8 +39,9 @@ echo $branch | grep release && tag_completed="${tag}"
 echo $branch | grep release || tag_completed="${tag}_${branch_short}"
 
 # We build the image
-echo "docker build -f $dockerFile -t $imageName:$tag_completed $context"
-docker build -f $dockerFile -t $imageName:$tag_completed $context
+echo "docker build -f "$dockerFile" -t "$imageName":"$tag_completed" $context"
```
Comment (albertdb) [inline scripts/pipelines/azure-devops/templates/package/quarkus-package.sh:42]:
```suggestion
echo docker build -f "$dockerFile" -t "$imageName:$tag_completed" "$context"
```

File: scripts/pipelines/azure-devops/pipeline_generator.sh [hunk 2]
```diff
@@ -219,8 +221,16 @@ function createPipeline {
     echo -e "${green}Generating the pipeline from the YAML template..."
     echo -ne ${white}
 
+    # This line go to the localDirectory of the repo and gets the repo name 
+    repoName="$(basename -s .git $(git config --get remote.origin.url))"
```
Comment (albertdb) [inline scripts/pipelines/azure-devops/pipeline_generator.sh:225]:
```suggestion
    repoName="$(basename -s .git "$(git config --get remote.origin.url)")"
```

**Review by albertdb — APPROVED** [review]

**Review by jbellver99 — APPROVED** [review]

**Pull request merged**